game/objects/Portal.gd

```diff
@@ -0,0 +1,6 @@
+extends Area2D
+
+export (String, FILE, "*.tscn") var next_scene: String
```

Comment:
I don't think we ever used a space between `export` and the `([...])` contents. I know it's a minor thing, but let's keep in line with the docs: `export(String, FILE, "*.tscn")`

Reply:
OK! Btw, is it OK to leave it blank, or better to fill a default value there?

Reply:
It's ok to be blank, the type matters most here. The dev. is in charge of setting the value to whatever he/she needs.